app/script/conversation/ConversationService.coffee

```diff
@@ -88,6 +88,18 @@ class z.conversation.ConversationService
   ###############################################################################
 
   ###
+  Remove bot from conversation.
+
+  @param conversation_id [String] ID of conversation to remove bot from
+  @param user_id [String] ID of bot to be removed from the the conversation
+  @param callback [Function] Function to be called on server return
+  ###
+  delete_bots: (conversation_id, user_id) ->
```

Comment:
Is it really `delete_bots` (Plural)? Because `user_id` can be just a String and not an Array.